src/Tests/dotnet-MsiInstallation.Tests/MsiInstallerTests.cs

```diff
@@ -0,0 +1,443 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.NET.Sdk.WorkloadManifestReader;
+
+namespace Microsoft.DotNet.MsiInstallerTests
+{
+    public class WorkloadTests : SdkTest, IDisposable
+    {
+
+        string SdkInstallerVersion
+        {
+            get
+            {
+                if (!string.IsNullOrEmpty(VM.VMTestSettings.SdkInstallerVersion))
+                {
+                    return VM.VMTestSettings.SdkInstallerVersion;
+                }
+                else
+                {
+                    return "8.0.101";
```

Comment:
Is there any method we have to get the current version of the .NET SDK to reduce the need to enter the sdk installer version in the json file in the future?

Reply:
This needs to be the version of the .NET SDK that's copied to `C:\SdkTesting` on the VM.  It doesn't necessarily need to be the same as the version we're testing.  So I don't think there's a good way to get it automatically right now.